Magento_PageBuilder/layout/default.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<page
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd"
+>
+    <body>
+        <referenceBlock name="contact-bar">
+            <arguments>
+                <argument name="custom-class" xsi:type="string">container width-full</argument>
+            </arguments>
+        </referenceBlock>
+        <referenceBlock name="footer.content">
+            <arguments>
+                <argument name="social-custom-class" xsi:type="string">pb-force-flex-row</argument>
```

Comment:
I'd use `pb` in name of this class to make it more clear that is added/available only when page builder is enable

Reply:
but footer template is available in all theme and You can set "social-custom-class" in any other place. This is something that we use in PB but it is not limited to that module so I'm not sure abut that :)

Reply:
For me would be clearer, because it's only in page builder module, this class isn't added when page builder is disable or in OS version, it could help some debugging, when you search classes globally or something IMO,  but it's not crucial, so if you don't see the need, we can leave it :)